src/main/java/mate/academy/spring/dao/UserDao.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.spring.dao;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+
+public interface UserDao {
+    User add(User user);
```

Comment:
```suggestion
    void add(User user);
```
По заданию вроде-бы он не должен ничего возвращать